cfgmgr/coppmgr.cpp

```diff
@@ -654,9 +654,6 @@ void CoppMgr::doCoppTrapTask(Consumer &consumer)
                                            m_coppTrapConfMap[key].trap_ids);
             }
 
-            m_coppTrapConfMap[key].trap_group = trap_group;
```

Comment:
Removing this will affect the logic inside addTrap since it is used to determine isTrapIdDisabled. I recommend instead to save the old trap group in a different variable and use it to compare below. You can remove the  assignment at the end of the if loop.
Please also add UT to cover this case